oxd-server/src/test/java/org/gluu/oxd/server/service/ValidatorTest.java

```diff
@@ -0,0 +1,67 @@
+package org.gluu.oxd.server.service;
+
+import org.gluu.oxauth.model.exception.InvalidJwtException;
+import org.gluu.oxauth.model.jwt.Jwt;
+import org.gluu.oxd.server.HttpException;
+import org.gluu.oxd.server.op.Validator;
+import org.testng.annotations.Test;
+
+import static org.testng.AssertJUnit.assertFalse;
+import static org.testng.AssertJUnit.assertTrue;
+
+public class ValidatorTest {
```

Comment:
Nice work! I like this test. One more missed test is `"aud": ["6b578a9b-7513-477a-9a7f-1343b487caf8"]` where `["6b578a9b-7513-477a-9a7f-1343b487caf8"]` is string, not parsed array. I know it shouldn't happen but it seems we got it sometime ago in #178, so better to handle it to be on safe side.

Reply:
Done